offix/src/main/java/org/aerogear/offix/interceptor/ConflictInterceptor.kt

```diff
@@ -4,11 +4,13 @@ import android.util.Log
 import com.apollographql.apollo.api.Mutation
 import com.apollographql.apollo.interceptor.ApolloInterceptor
 import com.apollographql.apollo.interceptor.ApolloInterceptorChain
+import org.aerogear.offix.interfaces.ConflictResolutionImpl
 import java.util.concurrent.Executor
 
-class ConflictInterceptor : ApolloInterceptor {
+class ConflictInterceptor(conflictResolutionImpl: ConflictResolutionImpl) : ApolloInterceptor {
```

Comment:
Make this as a val here so that you don't have to create an extra val down below.